src/globalStyles/newDataTableStyles.js

```diff
@@ -146,6 +146,36 @@ const newDataTableStyles = {
       justifyContent: 'center',
     },
   },
+  expiryBatchTextView: {
+    borderBottomColor: '#cbcbcb',
+    borderBottomWidth: 1,
+    flex: 1,
+    flexDirection: 'row',
+    width: '88%',
+    maxHeight: '65%',
```

Comment:
These percentage values trigger me lol

Reply:
I am open to suggestions on how better to recreate a text input underline !